pages/web-literacy/Skills.jsx

```diff
@@ -0,0 +1,88 @@
+var React = require('react');
+var HeroUnit = require('../../components/hero-unit.jsx');
+var Illustration = require('../../components/illustration.jsx');
+var OutboundLink = require('react-ga').OutboundLink;
+var skills = require('./skillsdata');
+var makeLinksFromWebLitSkills = require("./MakeLinksFromWebLitSkills.jsx");
+
+module.exports = React.createClass({
+  statics: {
+    pageTitle: '21st Century Skills',
+    pageClassName: 'web-literacy-skills-page'
+  },
+  generateSkills: function() {
+    return skills.map(function(skill) {
+      return (
+        <Illustration
+          width={150} height={150}
+          src1x={skill.imgSrc1x}
+          src2x={skill.imgSrc2x}
+          key={skill.name}
+          alt={"21c-skills-icon-" + skill.name}>
+          <h2 id={skill.name}>{skill.name}</h2>
+          <p>{skill.content}</p>
+          <h3>Competencies</h3>
+          <div className="competency-content">
+            { this.generateCompetencies(skill.competencies) }
+          </div>
+          <h3>Web Literacy Skills</h3>
+          {
+            Object.keys(skill.topics).map(function(topic) {
+              return (
+                <div><b>{topic}:</b> {makeLinksFromWebLitSkills(skill.topics[topic])}</div>
+              );
+            })
+          }
+        </Illustration>
+      );
+    }.bind(this));
+  },
+  generateCompetencies: function(competencies) {
+    competencies = competencies || [];
+    return competencies.map(function(competency) {
+      if (competency.name) {
+        return (
+          <div key={competency.name}>
+            <h4>{competency.name}</h4>
+            {
+              competency.content.map(function(contentItem) {
```

Comment:
this should probably be its own function, too. Inlining lots of maps makes things harder to follow.